README.md

```diff
@@ -15,7 +15,9 @@ Required nodejs v0.10.29+ and npm v2.1.0+.
 Configuration
 -------------
 
-TODO
+You can configure the client using the following environment variables:
+
+- `READINGLIST_SERVER_BASEURL`: the Readinglist server base URL; default: `http://0.0.0.0:8000/v0`
```

Comment:
In the furure we'll use envify, which will compile the js assets and alter them according to that env var. That way, deployment is mostly matter of building the project using the right vars set.